deploy/template.yaml

```diff
@@ -60,4 +66,10 @@ parameters:
   - name: AMQ_ADDRESS
     description: The URL to connect to an AMQ address
   - name: AMQ_QUEUE_NAME
-    description: The queue name to use for the AMQ address
\ No newline at end of file
+    description: The queue name to use for the AMQ address
+  - name: SERVER_URL
+    description: The RHSSO Server URL
+  - name: USER
+    description: The username for RHSSO login
+  - name: PWD
+    description: The Password for RHSSO login 
```

Comment:
Should we add RHSSO_ to all of these to make it more obvious what they are for? (RHSSO_SERVER_URL, RHSSO_USER, RHSSO_PWD etc...)

Reply:
Sounds good